scripts/inner/kubestellar

```diff
@@ -323,10 +341,45 @@ function ensure_espw() {
     else
         kubectl ws create "$espw_name" --enter
     fi
+    run_dex
+    sleep 2
+    run_kb_backend
+    kubectl apply -f "$bindir/../config/crds/"
+    kubectl get crd -oname | grep kubestellar.io | xargs -i kubectl label {} kube-bind.io/exported=true
```

Comment:
Please, let's just modify the YAML files in git.

Reply:
1. Wondering whether there could be use cases where these CRDs are not going to be kube-bound?
2. Will `make codegen` overwrite/delete the labels?
3. Also, I want to understand why it is better to modify the YAML files. Thanks!

Reply:
1. Use cases of KubeStellar as it is designed now? I think not.
2. No, `make codegen` does not run the crd-puller (which is what produced these YAML files).
3. My suggestion here is a shift-left play. It reduces what gets done at runtime.

Reply:
Re 2: sorry, I misunderstood the question. I was thinking of the CRDs in config/kube/crds/ .  For the CRDs in config/crds/ , yes, `make codegen` will overwrite them. We should make it put the desired added labels in. One way would be to provide the sort of patch that `hack/update-codegen-crds.sh` is already applying. Another might be an option in kubebuilder --- does it have a way to write a stylized comment in the IDL that adds a label to the CRD?

Reply:
Turns out that the kubebuilder option works. Kubebuilder supported this feature recently. I addressed this in PR https://github.com/kubestellar/kubestellar/pull/1354.